docker/snap-breakout/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.6.2
```

Comment:
It would be a good idea to provide a reference link to the version of the snap-atac pipeline that this script is designed to process in this file.